### PR TITLE
Create Raise the ESM Minimum Threshold - May 16, 2022.md

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
@prose11

### CODE REVIEW

File: governance/polls/Raise the ESM Minimum Threshold - May 16, 2022.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-05-16T16:00:00
+end_date: 2022-05-19T16:00:00
+---
+# Poll: Raise the ESM Minimum Threshold - May 16, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the $poll_source. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 16 at 16:00 UTC.
```
Comment (LongForWisdom) [inline governance/polls/Raise the ESM Minimum Threshold - May 16, 2022.md:18]: I guess MakerDAO Community, even though we did the signal.

File: governance/polls/Raise the ESM Minimum Threshold - May 16, 2022.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Raise the minimum threshold (`min`) to trigger the Emergency Shutdown Module (ESM) from 100,000 MKR to 150,000 MKR.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-increase-the-esm-minimum-threshold/15006) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows. 
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Friday executive passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
```
Comment (LongForWisdom) [inline governance/polls/Raise the ESM Minimum Threshold - May 16, 2022.md:37]: 'Friday Executive'

Comment (LongForWisdom) [reply governance/polls/Raise the ESM Minimum Threshold - May 16, 2022.md:37]: Will need changing in all the templates.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments.

**Review by prose11 — APPROVED** [review]
Nice work!

**Pull request merged**